componentLibraries/extensionLibrary/CMakeLists.txt

```diff
@@ -8,13 +8,15 @@ endif()
 
 set(targetname extensionlibrary)
 
+include(${CMAKE_CURRENT_LIST_DIR}/../../dependencies/fmilibrary.cmake)
```

Comment:
Is it possible to recompile extensionlibrary inside HopsanGUI ? otherwise you need to add bunch of compilation flags to the extensionlibrary.xml. Shold be documented somewhere how to do that.